src/Countries/US/NewMexico/NewMexicoWorkersCompensationEmployer/V20190101/NewMexicoWorkersCompensationEmployer.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Appleton\Taxes\Countries\US\NewMexico\NewMexicoWorkersCompensationEmployer\V20190101;
+
+use Appleton\Taxes\Countries\US\NewMexico\NewMexicoWorkersCompensationEmployer\NewMexicoWorkersCompensationEmployer as BaseNewMexicoWorkersCompensationEmployer;
+use Illuminate\Database\Eloquent\Collection;
+
+class NewMexicoWorkersCompensationEmployer extends BaseNewMexicoWorkersCompensationEmployer
+{
+    public function compute(Collection $tax_areas)
+    {
+        if ($this->payroll->getStartDate()->weekOfMonth === $this->payroll->getStartDate()->endOfQuarter()->weekOfMonth) {
+            return 2.30;
```

Comment:
Need to withhold tax here too.

Reply:
In this case no because this is an employer tax that isn't withheld.